Example/Swift Example/AppDelegate.swift

```diff
@@ -54,18 +54,18 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         application.registerForRemoteNotifications()
     }
 
-    func application(application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: NSData) {
+    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
         print("Successfully registered remote notification")
 
-        let skygear = SKYContainer.defaultContainer()
-        guard skygear.currentUser != nil else {
+        let skygear = SKYContainer.default()
+        guard skygear?.currentUser != nil else {
```

Comment:
I think it is quite weird that it becomes `skygear?`.

I just tested on xcode, the return type of `SKYContainer.default()` is  `SKYContainer!` so I expect it is not optional.

But it does not happen on other class, for example in `PubSubViewController`, you don't need to access `pubsubClient` with `?` and it is accessed with `SKYContainer.default().pubsubClient`.